### DISCUSSION TITLE
Vercel taking forever to load a page (TTFB)

### DISCUSSION THREAD

**Discussion by elie222** [vercel/vercel#6859] [Help]
I'm facing crazy long response times for a Next.js / Vercel app:

![image](https://user-images.githubusercontent.com/3090527/137791322-36485b8f-9bef-4627-ae90-0b331d45fd0d.png)

This is actually after a refresh of the page that it takes 23 seconds. The first attempt was pending forever.
If I refresh after it has successfully loaded it does load super quick though.

This is my code for `getStaticPaths`:

```tsx
export const getStaticPaths: GetStaticPaths = async () => {
  return { paths: [], fallback: 'blocking' };
};
```

There could be millions of paths so I'm intentionally returning an empty array. I will attempt to fix with `fallback: true` but I was under the impression that `blocking` was the preferred option from the docs.

**Comment by djclarkson** [comment]
Experiencing the same issue. Did you find a solution?

**Reply by elie222** [reply]
I don't remember. It somehow went away :(
We have both fallback: blocking and true for different pages in our codebase.
